_i18n/fr.yml

```diff
@@ -9,26 +9,36 @@ meta:
   contact:
     title: Contact
   contact_us:
+    description: Have a question or a problem with login.gov?
```

Comment:
still looks like english to me?

Reply:
...maybe the translations are still in English, because I ran my script on the English translations...